opensource.workers

```diff
@@ -1,4 +1,10 @@
 [
+	{
+                "name": "transfer_optim",
```

Comment:
can we use a complete name: `transfer_optimized` ?

Reply:
yes, I rename `transfer_optim` in `aws_cli` because it's not only link to transfer.